### PR TITLE
Adding new resources in Web Development file

### PR DESCRIPTION
**Pull request by shivam0110**

### CODE REVIEW

File: web-dev.md [hunk 1]
```diff
@@ -63,15 +64,25 @@
...
+- DevEd [JavaScript & SVG](https://www.youtube.com/channel/UClb90NQQcskPUGDIXsQEz5Q)
+
 ### Documentation
 
 - [JavaScript](https://developer.mozilla.org/en-US/docs/Web/javascript)
 - [React](https://reactjs.org/docs/getting-started.html)
 - [Angular](https://angular.io/docs)
 - [Node](https://nodejs.org/en/docs/)
 - [Deno](https://doc.deno.land/https/github.com/denoland/deno/releases/latest/download/lib.deno.d.ts)
+- [Chart.js](https://www.chartjs.org/docs/latest/)
```
Comment (thepushkarp) [inline web-dev.md:76]: Can we further divide Documentation into Framework and Libraries?

**Review by thepushkarp — CHANGES_REQUESTED** [review]

**Comment by yashshah2820** [comment]
@shivam0110 change the PR title to exactly what does the PR do, like what the changes it will produce to the repo

File: web-dev.md (hunk 1 above)
Comment (shivam0110) [reply web-dev.md:76]: Sure i will do it

**Comment by shivam0110** [comment]
> @shivam0110 change the PR title to exactly what does the PR do, like what the changes it will produce to the repo

Done

**Comment by thepushkarp** [comment]
Looks Good :sparkles:. Thanks

**Comment by shivam0110** [comment]
> Looks Good ✨. Thanks

Yeah, no problem

**Comment by thepushkarp** [comment]
One more thing, Bootstrap is a CSS Framework and not JS framework.

File: web-dev.md [hunk 2]
```diff
@@ -63,18 +66,41 @@
...
 - [Vue](https://vuejs.org/v2/guide/)
+- [Bootstrap](https://getbootstrap.com/docs/4.5/getting-started/introduction/)
+
+#### JS libraries 
+- [React](https://reactjs.org/docs/getting-started.html)
+- [jquery](https://api.jquery.com/)
+- [Chart.js](https://www.chartjs.org/docs/latest/)
+
+#### For Static Sites
+
```
Comment (yashshah2820) [inline web-dev.md:88]: remove this line

File: web-dev.md [hunk 3]
```diff
@@ -63,18 +66,41 @@
...
 - [JavaScript](https://developer.mozilla.org/en-US/docs/Web/javascript)
-- [React](https://reactjs.org/docs/getting-started.html)
-- [Angular](https://angular.io/docs)
 - [Node](https://nodejs.org/en/docs/)
 - [Deno](https://doc.deno.land/https/github.com/denoland/deno/releases/latest/download/lib.deno.d.ts)
+
+#### JS frameworks
+- [Angular](https://angular.io/docs)
 - [Vue](https://vuejs.org/v2/guide/)
+- [Bootstrap](https://getbootstrap.com/docs/4.5/getting-started/introduction/)
```
Comment (yashshah2820) [inline web-dev.md:80]: create a CSS framework and add this to it

**Comment by yashshah2820** [comment]
@shivam0110 create new PR for these changes

**Pull request merged**